src/Server/StandardServer.php

```diff
@@ -75,6 +75,7 @@ public function __construct($config)
             $config = ServerConfig::create($config);
         }
 
+        // @phpstan-ignore-next-line necessary until we can use proper union types
```

Comment:
I don't like adding to the baseline, because that does not allow adding a comment. Moving it over to the config file is pretty cumbersome and still does not allow pinpointing the exact location of the error.

There are cases for it, such as errors we can never fix (some variable property accesses), or bugs in PHPStan. In this case, I prefer the annotation. I think the risk of introducing a bug here and accidentally not noticing is very slim.